### DISCUSSION TITLE
expected release date of 0.6.0 version

### DISCUSSION THREAD

**Discussion by vinay1331** [JanusGraph/janusgraph#2720] [General]
What is the expected release date of 0.6.0 version

**Comment by porunov** [comment]
@vinay1331 see the comment here: https://github.com/JanusGraph/janusgraph/pull/2672#issuecomment-887680445

The proposed release day is August 2 (Monday) which will lead to voting process opening. If the release day isn't changed, the release will be available in 72 hours after the release voting process is started (i.e. August 5). Sometimes the release process may be delayed. I would generally look for about a week after the release voting is started (i.e. probably August 9 in average scenario).

**Reply by vinay1331** [reply]
thanks porunov

**Reply by vinay1331** [reply]
@porunov Any update on this. How soon we can expect this release.

**Reply by porunov** [reply]
We have a problem deploying  the new `janusgraph-grpc` module. I didn't work with that module much so can't say how long will it take to find a solution to fix the deployment. If you want to contribute your ideas or know how to correctly build and deploy grpc generated classes, let me know. The problem is described here: https://github.com/JanusGraph/janusgraph/pull/2672#issuecomment-895868763

**Reply by porunov** [reply]
The voting is now opened for 0.6.0 release at this location: https://lists.lfaidata.foundation/g/janusgraph-dev/topic/vote_janusgraph_0_6_0/84815207

**Reply by ghost** [reply]
what is the process of making a pre-release into a relese?
many thanks :)

**Reply by porunov** [reply]
Here is what goes after pre-release: https://github.com/JanusGraph/janusgraph/blob/master/RELEASING.md#create-a-vote-thread
Voting thread: https://lists.lfaidata.foundation/g/janusgraph-dev/topic/vote_janusgraph_0_6_0/84815207

**Comment by farodin91** [comment]
JanusGraph 0.6 is released.
